lib/grammar2.pegjs

```diff
@@ -72,7 +71,13 @@ ctrl = ifctrl
      / forctrl
      / returnctrl
 
-ifctrl     = "if" _ c:innerstmt _ b:block { return p.parseIf(c, b) }
+ifctrl = "if" _ c:innerstmt __ b:block ei:(__ elseifcont)* {
```

Comment:
For the `ei(__ elseifcont)*` does the * indicate that there can be an arbitrary amount of else if expressions? Just trying to learn more about writing PEGs.